tests/queries/0_stateless/01473_event_time_microseconds.sql

```diff
@@ -0,0 +1,25 @@
+-- This file contains tests for the event_time_microseconds field for various tables.
+-- Note: Only event_time_microseconds for asynchronous_metric_log table is tested via
+-- an integration test as those metrics take 60s by default to be updated.
+-- Refer: tests/integration/test_asynchronous_metric_log_table.
+
+set log_queries = 1;
+
+select '01473_metric_log_table_event_start_time_microseconds_test';
+system flush logs;
+SELECT sleep(3) Format Null;
```

Comment:
"Sleep" usage in functional tests is very impractible. Because we have lots of builds. For example with Thread Sanitizer (TSAN) program works 20x times slower. I think this test will work without sleep.

Reply:
Yes, I don't think it's necessary -- removed the sleep from here.